core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -105,14 +83,14 @@
    */
   public static final String FIELD_COUNT_PROPERTY_DEFAULT = "10";
 
-  int fieldcount;
+  private int fieldcount;
```

Comment:
But for proper inheritance you don't want to use getters/setters, you want to access the variables directly (plus avoid the function call [which would eventually be inlined, but...])